core/src/main/java/hudson/model/Computer.java

```diff
@@ -1411,7 +1411,7 @@ public Api getApi() {
      */
     public void doDumpExportTable( StaplerRequest req, StaplerResponse rsp ) throws IOException, ServletException, InterruptedException {
         // this is a debug probe and may expose sensitive information
-        checkPermission(Jenkins.ADMINISTER);
```

Comment:
ublic void doDumpExportTable( StaplerRequest req, StaplerResponse rsp ) throws IOException, ServletException, InterruptedException {
        // this is a debug probe and may expose sensitive information
        checkPermission(Jenkins.ADMINISTER);